install.sh

```diff
@@ -231,7 +239,7 @@ user_provision:
   groups:
     - bypass2fa
   ssh_keys:
-    - "{{ lookup('file', '/home/${CONTROLLER_USER}/ce-provision/data/localhost/home/${CONTROLLER_USER}/.ssh/id_ecdsa.pub') }}"
+    - "{{ lookup('file', '/home/${CONTROLLER_USER}/ce-provision/data/localhost/home/${CONTROLLER_USER}/.ssh/id_ed25519.pub') }}"
```

Comment:
Should this use `public_key_name`, rather than hardcoding `id_ed25519.pub`?

Reply:
That can work in the installer, you're right - but we can't generally do it, because `public_key_name` belongs to the `ce_provision` role and it's unlikely both that role and the `user_provision` role will be available on the same machine, most machines loading the `user_provision` role will not have the `ce_provision.public_key_name` variable loaded in. We'd have to move it to the `_init` role defaults to make it always available.

Reply:
Made the change, just waiting to CI to make sure it doesn't break the installer. :smiley:

Reply:
Fixed, CI green! :partying_face: